modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-third-party-monitoring-web-service-apis"]
+= Accessing third-party monitoring web service APIs
+
+[role="_abstract"]
+You can directly access third-party web service APIs from the command line for monitoring stack components such as Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier. 
```

Comment:
no need to mention Prometheus here, since Prometheus UI is still could be accessed in 4.10

Reply:
> no need to mention Prometheus here, since Prometheus UI is still could be accessed in 4.10

Prometheus API is also accessible.

Reply:
I'm keeping the reference to Prometheus because it's API is also accessible.